### PR TITLE
migrate `sig-node` jobs to community cluster

### PR DESCRIPTION
**Pull request by rjsadow**
This PR moves sig-node jobs to the community owned cluster gke cluster

ref: https://github.com/kubernetes/test-infra/issues/30277

/cc @mrunalp @klueska @SergeyKanzhelev @endocrimes

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/k8s-release-branches.yaml
Comment (kannon92) [inline config/jobs/kubernetes/sig-node/k8s-release-branches.yaml:3]: You maybe have to remove the project type as that is not defined in the new cluster.

**Comment by kannon92** [comment]
As I have commented, you should remove the gcp-project-type from these jobs.

https://github.com/kubernetes/test-infra/pull/30957/files

we learned the hard way that leaving those in will cause jobs to fail as that is not defined in the new cluster.  I think most of your jobs that you converted still have this project type.

File: config/jobs/kubernetes/sig-node/k8s-release-branches.yaml
Comment (rjsadow) [reply config/jobs/kubernetes/sig-node/k8s-release-branches.yaml:3]: Updated.

**Comment by kannon92** [comment]
/lgtm
/assign @SergeyKanzhelev @dims 

@rjsadow once this gets merged, can you make sure to monitor these jobs and make sure they are all working?  Some of these jobs are not currently working so I'd suggest looking over the ones you changed

**Comment by kannon92** [comment]
/hold

Since these impact the release-branch, we should probably wait until k8s/k8s opens up again.

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
